TelligentLocalInstance/TelligentLocalInstance.ps1

```diff
@@ -240,6 +237,51 @@ function Get-CommunitySolrVersion {
 	}
 }
 
+function Get-CommunitySolrUrl {
+    <#
+        .SYNOPSIS
+            Gets the Solr version for a given community version numeber
+        .PARAMETER Version
+            The community version
+        .EXAMPLE
+            Get-CommunitySolrVersion 9.0
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true, ValueFromPipelineByPropertyName=$true)]
+        [ValidateNotNullOrEmpty()]
+        [Version]$Version
+    )
+    $solrVersion = Get-CommunitySolrVersion $Version
```

Comment:
We do because eventually the port number will be dynamic depending on the Solr version so this is kind of a placeholder.